client/src/themes/theme.js

```diff
@@ -9,6 +9,9 @@ export const theme = createMuiTheme({
     }
   },
   palette: {
-    primary: { main: "#DF1B1B" }
+    primary: { main: "#DF1B1B" },
+    background: {
+      default: "#fbfcff"
+    }
   }
 });
```

Comment:
Good job! and I have a small suggestion here.
Latter, if we still need to change UI, may we can take the advantages of theme.js. If we override the default style here, new style will be applied to all the components. https://material-ui.com/customization/theming/ 
For example, if we put 
`overrides: {
    MuiButton: {
      root: {
        borderRadius: '30px',
      },
    },
  },` into `createMuiTheme` like you overrode the palette background here, all the buttons will have round corners. We don't need to modify every button in every file.